src/wizard/Wizard.tsx

```diff
@@ -76,92 +21,39 @@ export interface WizardStep {
 }
 
 export interface WizardProps {
+    className?: string;
     initialStepKey: string;
-    onStepChangeRequest: (
+    useSnackFeedback?: boolean;
+    lastClickableStepIndex?: number;
+    steps: WizardStep[];
+    NavigationComponent?: (props: NavigationProps) => ReactElement | null;
+    StepperComponent?: (props: StepperProps) => ReactElement | null;
+    onStepChangeRequest?: (
         currentStep: WizardStep,
         newStep: WizardStep
     ) => Promise<string[] | undefined>;
     onStepChange?: (stepKey: string) => void;
-    useSnackFeedback?: boolean;
-    steps: WizardStep[];
-    lastClickableStepIndex?: number;
 }
 
 export const Wizard: React.FC<WizardProps> = ({
+    className,
     initialStepKey,
-    onStepChangeRequest,
-    onStepChange = _.noop,
+    onStepChangeRequest = () => undefined,
```

Comment:
This default assignment erases any typing for the function (onStepChangeRequest is now `any`), we need some other way.